### PR TITLE
code restructured and image url build

### PR DESCRIPTION
**Pull request by hupling**
add a customized picture for a recipe

### CODE REVIEW

File: app/src/main/java/team3/recipefinder/MainActivity.kt [hunk 1]
```diff
@@ -95,24 +96,25 @@ class MainActivity : AppCompatActivity(), CreateRecipeFragment.CreateRecipeListe
...
     }
 
     /**
-     * Method that handles the negativeClick for the different dialogs.
+     * Method that habdeles the negativeClick for create Ingredient
      */
-    override fun onDialogNegativeClick() {
+    override fun saveItem(id: String?, name: String?) {
+        Toast.makeText(this,  "Needs to be impelemented",Toast.LENGTH_LONG).show();
     }
```
Comment (hupling) [inline app/src/main/java/team3/recipefinder/MainActivity.kt:118]: The add ingredient function in the main activity doesn't work. Maybe we should remove it

**Comment by bremade** [comment]
LGTM tested locally. Well just add the edit ingredient picture and add ingredient from toolbar menu.
As discussed in the meeting.

File: app/src/main/java/team3/recipefinder/MainActivity.kt (hunk 1 above)
Comment (hupling) [reply app/src/main/java/team3/recipefinder/MainActivity.kt:121]: solved

**Comment by joaosousa98** [comment]
I'm not sure but 2 checks were not successful. How bad are those failures?

**Pull request merged**